exercises/practice/order-management/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "The classical introductory exercise. Just say \"Hello, World!\".",
+  "blurb": "Deprecated",
```

Comment:
Could you find the original blurb of the exercise? It would not matter for new students, but it would for older students.